app/controllers/settings/exports/lists_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Settings
   module Exports
-    class ListsController < ApplicationController
+    class ListsController < BaseController
```

Comment:
ok, if this inherits from Settings::BaseController, doesn't that still require_functional!?